R/impute.R

```diff
@@ -12,8 +12,8 @@
 #'
 #' @param draws A `draws` object created by [draws()].
 #'
-#' @param references a named vector. Identifies the references to be used when generating
-#' the imputed values. Should be of the form `c("Group1" = "Reference1", "Group2" = "Reference2")`.
+#' @param references a named vector. Identifies the references to be used when for reference-based 
```

Comment:
Maybe we can delete "when" from this sentence?

Reply:
Done